examples/eth1_del_all_route_rule.yml

```diff
@@ -0,0 +1,5 @@
+---
+interfaces:
+  - name: eth1
```

Comment:
Is it intended that the interface needs to be specified? Seems like a bug to me, what do you think?

Reply:
Yes. I am been working on this issue in PR #488